templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -3,9 +3,19 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/keepalived.conf.tmpl"
 contents:
   inline: |
     vrrp_script chk_ocp {
-        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:6443/readyz && /usr/bin/curl -o /dev/null -kLfs http://localhost:50936/readyz"
+        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:9443/readyz"
```

Comment:
IIRC, LB port pass as a parameter in baremetal-runtimecfg  default to 9443, I assume it will be better to render it

[1] https://github.com/openshift/baremetal-runtimecfg/blob/master/cmd/monitor/monitor.go#L55

Reply:
Good point. Will update.